pkg/api/router.go

```diff
@@ -63,7 +78,10 @@ func (s *server) setupRouting() {
 	})
 
 	handle(router, "/bzz/{address}/{path:.*}", jsonhttp.MethodHandler{
-		"GET": http.HandlerFunc(s.bzzDownloadHandler),
+		"GET": web.ChainHandlers(
+			s.newTracingHandler("bzz-download"),
+			web.FinalHandlerFunc(http.HandlerFunc(s.bzzDownloadHandler)),
```

Comment:
`http.HandlerFunc` can be dropped in all changes in router.

```suggestion
			web.FinalHandlerFunc(s.bzzDownloadHandler),
```